modules/serverpod_auth/serverpod_auth_sms_flutter/test/serverpod_auth_sms_flutter_test.dart

```diff
@@ -0,0 +1 @@
+//Todo: Write test
```

Comment:
Chore: This code needs tests.

I'm working on a PR now that might help set the expectation around what tests are required: https://github.com/serverpod/serverpod/pull/1993